src/algebra/order/with_zero.lean

```diff
@@ -117,6 +117,9 @@ le_of_le_mul_right h (by simpa [h] using hab)
 lemma mul_inv_le_of_le_mul (h : c ≠ 0) (hab : a ≤ b * c) : a * c⁻¹ ≤ b :=
 le_of_le_mul_right h (by simpa [h] using hab)
 
+lemma mul_inv_le_iff₀ (hb : b ≠ 0) : a * b⁻¹ ≤ c ↔ a ≤ c * b :=
+⟨λ h, inv_inv₀ b ▸ le_mul_inv_of_mul_le (inv_ne_zero hb) h, mul_inv_le_of_le_mul hb⟩
```

Comment:
can you add a version of this for `a ≤ b * c⁻¹ ↔ a * c ≤ b`? It should be the same but use `mul_inv_le_of_le_mul` instead

Reply:
Sure! It should be named le_mul_inv_iff₀, right?

Reply:
Done; I added `le_div_iff₀ (hc : c ≠ 0) : a ≤ b/c ↔ a*c ≤ b` as well.